packages/react-charting/UnitTests/VerticalBarChartUT.test.tsx

```diff
@@ -25,13 +25,11 @@ const stringPoints = [
   },
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30771&commitId=7c5a24a5012633d9e37805b25cafcb97ff749d04&env=prod&clientType=FLUENTUI)


  <details><summary><b>react-charting-AreaChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-AreaChart.Custom Accessibility.default.chromium.png|11| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

Reply:
VR link is not working. Resolving the comment as the change is only in test files and not in the product.